### PR TITLE
Feature/create audio manager

### PR DESCRIPTION
**Pull request by Corvus-inc**

### CODE REVIEW

File: Assets/Scripts/Sounds/SoundAssets.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using UnityEngine;
+
+namespace Sounds
+{
+    public class SoundAssets : MonoBehaviour
+    {
+        [SerializeField] private SoundAudioClip[] soundAudioClipArray;
+
+        public SoundAudioClip[] SoundAudioClipArray => soundAudioClipArray;
+        public static SoundAssets I
```
Comment (pushkinman) [inline Assets/Scripts/Sounds/SoundAssets.cs:10]: Why the name 'I'?

Comment (Corvus-inc) [reply Assets/Scripts/Sounds/SoundAssets.cs:10]: I wanted formulated the question for this place and corrected that. But i forgot this.

Comment (Corvus-inc) [reply Assets/Scripts/Sounds/SoundAssets.cs:10]: Fix it now?

Comment (Corvus-inc) [reply Assets/Scripts/Sounds/SoundAssets.cs]: fixed

**Review by pushkinman — APPROVED** [review]

**Pull request merged**